front-end/src/components/SearchResult.vue

```diff
@@ -124,9 +130,9 @@ const downloadLabel = computed(
           focusable="false"
           role="img"
           class="usa-icon"
-          :aria-labelledby="downloadLabel"
+          :aria-label="downloadLabel"
```

Comment:
If this is being switched from the aria-labelledby with an ID of "downloadLabel" to just uses the standard aria-label with a string value, then it should be changed to include the same content as the string that is being displayed in the button.